### DISCUSSION TITLE
While applying for github student pack it says there are outstanding issues with your GitHub account. Please contact GitHub Support for more information.

### DISCUSSION THREAD

**Discussion by ridafatima90** [community/community#62721] [GitHub Education]
### Hi everyone,

Hello:
I am writing to you today to request assistance with applying for the GitHub Student Pack. I am a student at Stris, and I am interested in using the GitHub Student Pack to access the many benefits that it offers, such as free private repositories, GitHub Education Pack, and GitHub Student Developer Pack.

However, when I try to apply for the GitHub Student Pack, I am unable to do so because there is an issue with my GitHub account. Specifically, I am unable to verify my student status. I have tried to verify my student status using the upload documents
 but I have been unsuccessful.

I have attached a copy of my student ID to this email. I have also tried to contact GitHub support, but I have not received a response.

I would be grateful if you could help me to resolve this issue so that I can apply for the GitHub Student Pack. I am a software engineering student, and I am very interested in using GitHub to learn and collaborate with other developers. I believe that the GitHub Student Pack would be a valuable resource for me.

Thank you for your time and consideration.

**Comment by sssst9s** [comment]
Hey !

In regard to the GitHub Student Pack, I understand that navigating its various components can sometimes be complex due to the multitude of available options. Allow me to offer some guidance that may assist you in your endeavor:

**Reapplying and Re-uploading:** In some cases, reapplying or re-uploading your information might help resolve issues. Occasionally, technical glitches can affect the process, and a subsequent attempt may lead to success.

**School Eligibility Verification:** Prior to proceeding, I recommend verifying whether your educational institution is officially recognized and meets the eligibility criteria for participation in the GitHub Student Program. This information can often be found during the application process itself.

**Engaging GitHub Support:** Should you encounter persistent difficulties, reaching out to GitHub's dedicated support team is a prudent step. Utilize the following link to establish contact and initiate a support ticket: [GitHub Support](https://support.github.com/contact).

Additionally, if you have recently created your GitHub account, it's possible that certain security measures could flag the account. While these measures are in place to ensure the integrity of the platform, they may inadvertently affect legitimate applications. Should this be the case, the GitHub support team will swiftly assist you in resolving any such issues.

Hope it helps a bit 🚀

**Reply by ridafatima90** [reply]
Hi,
Please refer to the attached. It says while applying There are outstanding issues with your GitHub account. Please contact [GitHub Support](https://support.github.com/contact?tags=education-support) for more information. I am not able to find the ways to resolve this issue. And this has beeen pending from long time. Earlier i raised ticket also but it was closed without any resolution. Any help would be highly appreciated. Here is the link https://support.github.com/ticket/personal/0/2130696. ticket id is 2130696

![image](https://github.com/community/community/assets/128382099/d6a46580-d1f0-44df-816e-e8ce20addc66)


Thank you

**Comment by sssst9s** [comment]
In reference to the screenshot you've shared, it appears that the challenge you're encountering could potentially be associated with your GitHub account settings. A likely scenario is that the issue might stem from the email verification process. To proceed with the application for the GitHub Student Pack, it's essential to confirm your email address. While this step may seem routine, it serves as a fundamental prerequisite for eligibility.

Furthermore, I'd like to draw your attention to the possibility that the concern could be attributed to the recency of your GitHub account. Accounts that have been recently created might encounter certain limitations or restrictions. In such cases, the most effective approach for resolving this matter would be to initiate a support ticket. In the event that a previously created ticket has been closed without a clear resolution, I recommend reopening the ticket and detailing the nature of the issue. The responsiveness of the support team can vary based on their current workload.

Also it's essential to ensure that your educational institution qualifies for participation in the GitHub Student Pack. To confirm eligibility, I recommend reviewing the list of eligible schools provided in the GitHub documentation. You can access this information at the following link: [GitHub Student Pack Eligibility](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student).
[Contact Github Support](https://support.github.com/request)

**Reply by ridafatima90** [reply]
In reference to the screenshot you've shared, it appears that the challenge you're encountering could potentially be associated with your GitHub account settings. A likely scenario is that the issue might stem from the email verification process. To proceed with the application for the GitHub Student Pack, it's essential to confirm your email address. While this step may seem routine, it serves as a fundamental prerequisite for eligibility. **Email is confirmed via my account long time ago**

Furthermore, I'd like to draw your attention to the possibility that the concern could be attributed to the recency of your GitHub account. Accounts that have been recently created might encounter certain limitations or restrictions. In such cases, the most effective approach for resolving this matter would be to initiate a support ticket. In the event that a previously created ticket has been closed without a clear resolution, I recommend reopening the ticket and detailing the nature of the issue. The responsiveness of the support team can vary based on their current workload. **Account is i think of more than 3-4 months. Raised again a ticket 2280422** 

Also it's essential to ensure that your educational institution qualifies for participation in the GitHub Student Pack. To confirm eligibility, I recommend reviewing the list of eligible schools provided in the GitHub documentation. You can access this information at the following link: **My school is registered. it is showing in dropdown list**

In the past i have been travelling to other countries and have applied from there as well. I have searched on internet for similar kind of message. Link here https://github.com/orgs/community/discussions/23109
But as per the above link i could not find any sanction notification in my account. Could you please provide any direction on that?

Thank you!

**Comment by ridafatima90** [comment]
Hi:
Is there anyone who can properly guide me what is the issue here ? I am trying to apply from past 3-4 months but hasn't succeeded yet. Any concrete advice relate to my issue would be higly appreciated.!!!

**Comment by DSDmark** [comment]
Hi @ridafatima90,

Please don't upload your personal document here.

**Reply by ridafatima90** [reply]
Noted done. Can you assist me further on that, if you know the resolution. Thanks!

**Comment by zhangchi25806** [comment]
Same here, still waiting for help too

**Reply by ElkhanAbbasov** [reply]
any news?

**Comment by IsmaelE70** [comment]
i also have same problem 
"There are outstanding issues with your GitHub account. Please contact [GitHub Support](https://support.github.com/contact) for more information."

**Reply by ElkhanAbbasov** [reply]
any news?

**Reply by ShafiulKabirS** [reply]
any update?

**Reply by BakulBd** [reply]
Any update? Anyone get any solutions?
